### PR TITLE
More speed.

### PR DESCRIPTION
**Pull request by kwagyeman**
* Not done yet...

### CODE REVIEW

File: src/omv/ov5640.c [hunk 1]
```diff
@@ -889,7 +897,7 @@ int ov5640_init(sensor_t *sensor)
     sensor->set_lens_correction = set_lens_correction;
 
     // Set sensor flags
-    SENSOR_HW_FLAGS_SET(sensor, SENSOR_HW_FLAGS_VSYNC, 0);
+    SENSOR_HW_FLAGS_SET(sensor, SENSOR_HW_FLAGS_VSYNC, 1);
```
Comment (iabdalkader) [inline src/omv/ov5640.c]: Are you sure about this ? If VSYNC was wrong, it shouldn't have worked... @kwagyeman

**Comment by iabdalkader** [comment]
@kwagyeman please use a more descriptive commit message, also do you want this merged now ?

File: src/omv/ov5640.c (hunk 1 above)
Comment (kwagyeman) [reply src/omv/ov5640.c]: I changed a register that was inverting the vsync signal too.

**Comment by kwagyeman** [comment]
It doesn't have to be merged yet. But, was pushing it for others to see. There's no harm merging it however.

File: src/omv/ov5640.c [hunk 2]
```diff
@@ -40,14 +40,14 @@ static const uint8_t default_regs[][3] = {
 
 // https://github.com/ArduCAM/Arduino/blob/master/ArduCAM/ov5640_regs.h
 
-    { 0x47, 0x40, 0x21 }, // { 0x47, 0x40, 0x20 },
```
Comment (kwagyeman) [inline src/omv/ov5640.c]: This bit was inverting vsync.

**Pull request closed without merging**